srcpkgs/itch/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'itch'
+pkgname=itch
+version=1.18.0
+revision=1
+archs="i686 x86_64"
+wrksrc="${pkgname}-setup-${version}"
+build_style=go
+go_import_path="github.com/itchio/itch-setup"
+hostmakedepends="git pkg-config"
+makedepends="gtk+3-devel"
+short_desc="Itch.io indie games client for Linux"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="MIT"
+homepage="https://itch.io/"
+distfiles="
+ https://github.com/itchio/itch-setup/archive/v${version}.tar.gz
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon16.png
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon32.png
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon48.png"
+checksum="
+ 96d3bff9b200dc5c03af8e46d0805a6f4dc504d6281b2956dcedda931849c0dc
+ c1e7154020c847cd5cd75d544a7e5dbe8bd495fe87a664afd24c4a952a7e8a78
+ e38c4a04b2ced31d2a13b72f310e069beff73a7cf887a4cbde57fa7b19c67fab
+ 5f38d0b884053b10d856151931b7699723c4a28cf14622c95bc6022bec391c3f"
+skip_extraction="icon16.png icon32.png icon48.png"
+
+do_build() {
+	cd "$GOSRCPATH"
+	go get -p "$XBPS_MAKEJOBS" -x -tags "${go_build_tags}" -ldflags "${go_ldflags}" ${go_package}
+}
```

Comment:
Why can't this use the `do_build` function from the Go build style?

Reply:
In this version 1.18.0 you needed it, now in version 1.24.0 you don't need it, I will update the template and make squash the commits.